### DISCUSSION TITLE
RdzTTgo Remote Station

### DISCUSSION THREAD

**Discussion by Mauri740** [dl9rdz/rdz_ttgo_sonde#373] [Q&A]
hello everyone, I would like to access my Rdz ttgo used in a remote station located at a high altitude. The project includes a 4G WiFi router with a dynamic IP powered by  battery recharged by a solar panel, the 4G network coverage in that area is good. I thinking to use a Raspberry Pi Zero and remotely connected by another PC via a DNS system. Is there anything simpler? Thank you

**Comment by DHSsonde** [comment]
I do something similar with a raspberry pi 4. I use a paid VPN (nordVPN) with meshnet so it's secure. There are free alternatives to that as well. I do not have experience with the hardware limits of the pi zero because I do not have one. 

From the sounds of it you are looking into the dynamic DNS  route. From my experience I did not want to do that because of security reasons which you may find in your research with that method. 

Check out Twingate.com they offer a free option and they do say it supports raspberry pi.

**Comment by lelun** [comment]
i have a different remote system deployed but it has mains power. all it dues is report to sondehub and is a simple TTGO and 4G wifi and the TTGO is set to mode 1 that will auto conect it to the wifi incase it looses conection, minimal maintenence unless a lightning strike kills it. 

nr 2 system that i am setting up to really up the coverage is a raspberry pi 3 with a SDR and a TTGO as a backup. all systems either are or will use 433 Mhz short magnetic antennas. due to there good range and omnidirectionality. main issue is weather proofing imo.

but if you only intend to upload data to sondehub a TTGO with mode 1 and a router is all that is needed. defualt mode is mode 3.
